build/linux/installer/conf/fluent-bit.conf

```diff
@@ -54,5 +54,5 @@
 #${KubernetesFilterEnabled}    Kubelet_Port        10250
 #${KubernetesFilterEnabled}    Kubelet_Host        ${NODE_IP}
 #${KubernetesFilterEnabled}    tls.verify          Off
-#${KubernetesFilterEnabled}    K8S-Logging.Exclude On
+#${KubernetesFilterEnabled}#${AnnotationBasedLogFilteringEnabled}    K8S-Logging.Exclude On
```

Comment:
is there any space needed for the two commented out section? fluent-bit might raise some error if the dent is not right